### PR TITLE
adjust a few error messages

### PR DESCRIPTION
**Pull request by lewgun**
### What problem does this PR solve? 
adjust a few error messages

### What is changed and how it works?
only adjust a few error messages, don't change any logic

### Check List 

Tests 

 - Unit test

Code changes

- N/A

Side effects

- N/A

Related changes
- N/A

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@lewgun  thanks, LGTM

File: pkg/column-mapping/column.go [hunk 1]
```diff
@@ -507,13 +507,13 @@ func computePartitionID(schema, table string, rule *Rule) (instanceID int64, sch
...
 	if len(prefix) >= len(name) || prefix != name[:len(prefix)] {
-		return 0, errors.NotValidf("%s is not prefix of %s", prefix, name)
+		return 0, errors.NotValidf("%s is not the prefix of %s", prefix, name)
 	}
 
 	idStr := name[len(prefix):]
 	id, err := strconv.ParseUint(idStr, 10, bitSize)
 	if err != nil {
-		return 0, errors.NotValidf("suffix of %s is not int64", idStr)
+		return 0, errors.NotValidf("suffix of %s's data type is not int64", idStr)
```
Comment (csuzhangxc) [inline pkg/column-mapping/column.go]: we should focus on `idStr`, can we change the statement not followed by `'s`?

Comment (lewgun) [reply pkg/column-mapping/column.go]: like this :"suffix of the idStr, can't be converted to int64" ?

Comment (csuzhangxc) [reply pkg/column-mapping/column.go]: `suffix of the idStr can't be converted to int64`?

Comment (lewgun) [reply pkg/column-mapping/column.go]: LGTM

**Comment by lewgun** [comment]
PTAL @csuzhangxc

**Review by csuzhangxc — APPROVED** [review]
LGTM

**Pull request merged**